CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Release 2.27.0 02-06-2021
+
+- b/184731851: Fix overhead latency calculation for backend timeout (#505)
+- b/188906398: Reduce backoff initial latency. (#520)
+- b/188808823: Simplify CORS regex and fix brittle tests (#519)
+- b/188583118: Update googleapis/googleapis for golang code (#516)
```

Comment:
Remove the bug ids.  They don't mean anything to external users.

Reply:
Done.